app/lib/submission_builder/ty2022/states/ny/individual_return.rb

```diff
@@ -213,9 +213,8 @@ def supported_documents
 
             supported_docs << {
               xml: nil,
-              pdf: PdfFiller::AdditionalDependentsPdf,
+              pdf: SubmissionBuilder::Ty2022::States::Ny::Documents::It201AdditionalDependents,
```

Comment:
Was the AdditionalDependentsPdf not working at all?

Reply:
It has different fields, accepts different input, and is all in all just a different case. So I decided to change what was used.